src/main/java/com/google/sps/servlets/NewProjectServlet.java

```diff
@@ -22,27 +22,47 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+//For saving the user who creates a project
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
 
 /** Servlet responsible for creating new projects. */
 @WebServlet("/post-project")
 public class NewProjectServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String title = request.getParameter("title");
-    String description = request.getParameter("description");
-    long timestamp = System.currentTimeMillis();
-    String giturl = request.getParameter("giturl");
-
+    
+    long timestamp = System.currentTimeMillis();                    //store post time
+    //UserService userService = UserServiceFactory.getUserService();  //store creator
+    //String email = userService.getCurrentUser().getEmail();
+    //general project info
+    String title = request.getParameter("title");                   //store name of project
+    String description = request.getParameter("description");       //store description text
+    String giturl = request.getParameter("giturl");                 //store git url 
+    //tags
+    String language = request.getParameter("tag-languages");        //store language
+    String timezone = request.getParameter("tag-timez");             //store time-zone
+    String ratediff = request.getParameter("difficulties");         //store difficulties
+    String timecommit = request.getParameter("time-commitment");    //store time commitment
+    String collabtyp = request.getParameter("collab-type");         //store collaboration method
+    
     Entity projectEntity = new Entity("Project");
+    //projectEntity.setProperty("owner_email", email);
+    projectEntity.setProperty("timestamp", timestamp);
```

Comment:
because you only use these variables once, you don't need to first declare them as variables, i.e. just do:

```
projectEntity.setProperty("title", request.getParameter("title"));
```